src/payment/strategies/amazon-pay/amazon-pay-payment-strategy.spec.ts

```diff
@@ -9,23 +9,41 @@ import { BillingAddressActionType } from '../../../billing/billing-address-actio
 import { getBillingAddress, getBillingAddressState } from '../../../billing/billing-addresses.mock';
 import { createCheckoutStore, CheckoutRequestSender, CheckoutStore, CheckoutStoreState, CheckoutValidator } from '../../../checkout';
```

Comment:
Could you please split this import: 
```suggestion
import {
createCheckoutStore,
CheckoutRequestSender,
CheckoutStore,
CheckoutStoreState,
CheckoutValidator
} from '../../../checkout';
```

I know you didn't modify this line but we also can upgrade the quality code, we split the import when there are 3+ components